src/cformat.jl

```diff
@@ -1,16 +1,80 @@
 formatters = Dict{ Compat.ASCIIString, Function }()
 
-function sprintf1( fmt::Compat.ASCIIString, x )
+@static if VERSION >= v"0.6.0-dev.1671"
```

Comment:
Although, in these cases where I've used it to distinguish between v0.5.x and v0.6, both branches are being tested and used (with Travis-CI on GitHub, and at work, where even though we are deploying now on v0.5.1, I'm making sure that we'll be ready for v0.6 as soon as it is released and all of the packages that we use have been updated to not have any deprecations)